commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerCommand.java

```diff
@@ -172,6 +172,31 @@
       arity = "1")
   private final FilecoinNetwork filecoinNetwork = FilecoinNetwork.TESTNET;
 
+  @Option(
+      names = {"--slashing-storage-url"},
+      description = "A string defining the storage mechanism for slashing data eg. sql:jdbcurl",
+      paramLabel = "<storage label>",
+      arity = "1")
+  private String slashingStorageUrl = "leveldb"; // not sure on default - not Level!
+
+  @Option(
+      names = {"--slashing-protection-enabled"},
```

Comment:
Do we need to have this option specific to eth2? When filecoin slashing exists we probably want to be able to just the filecoin slashing protection not the eth slashing protection.